modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js

```diff
@@ -202,88 +299,156 @@ function EditItem() {
 				</div>
 
 				<div className="flex-grow-1 flex-shrink-1 overflow-auto p-4">
-					{options?.items?.length > 0 ? (
-						<ClayForm>
-							<Options
-								cartItemId={editedItem.cartItemId}
-								channelId={channel.id}
-								namespace={MINI_CART_NAMESPACE}
-								productId={editedItem.productId}
-								productOptions={options.items}
-								selectedItem={selectedItem}
-							/>
-						</ClayForm>
+					{Liferay.FeatureFlags['COMMERCE-9599'] &&
+					options?.items?.length > 0 ? (
+						<>
+							<div className="panel panel-unstyled">
+								<div className="panel-header">
+									<span className="panel-title">
+										{Liferay.Language.get('edit-options')}
+									</span>
+								</div>
+							</div>
+
+							<ClayForm>
+								<Options
+									cartItemId={editedItem.cartItemId}
+									channelId={channel.id}
+									namespace={MINI_CART_NAMESPACE}
+									productId={editedItem.productId}
+									productOptions={options.items}
+									selectedItem={selectedItem}
+								/>
+							</ClayForm>
+						</>
 					) : null}
 
-					{priceOnApplication && (
-						<div className="mini-cart-prices mt-4">
-							<PriceRow
-								priceName={Liferay.Language.get(
-									'price-as-configured'
-								)}
-							>
-								<span className="price-on-application price-value text-3">
-									{Liferay.Language.get(
-										'price-on-application'
-									)}
-								</span>
-							</PriceRow>
-						</div>
+					{cpInstance.id ? (
+						<>
+							<div>
+								<div className="panel panel-unstyled">
+									<div className="panel-header">
+										<span className="panel-title">
```

Comment:
this looks like there will be 2 panels 1 for edit quantity and 1 for edit UOM how is it displayed this?

Reply:
Yes, we will have separate panels for them as we agreed with the design team. I reported it here as well: https://liferay.atlassian.net/browse/COMMERCE-12316?focusedCommentId=2559868